src/main/java/com/google/api/codegen/DiscoveryFragmentGeneratorTool.java

```diff
@@ -57,6 +57,13 @@ public static void main(String[] args) throws Exception {
             .hasArg()
             .argName("OUTPUT-DIRECTORY")
             .build());
+    options.addOption(
```

Comment:
This is totally fine, but consider whether in the future a YAML input format would be better. If so, we can plan for it.